### PR TITLE
feat: add goreleaser config for brew and docker

### PR DESCRIPTION
**Pull request by GrayFlash**

### CODE REVIEW

**Comment by GrayFlash** [comment]
@StewartJingga can you verify it once, locally it worked fine. Also it will be a better idea to update the name and email id in commit author for brews with yours as I will not have the access to the repo and the release will fail later when initiated by ci.

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,5 @@
...
+FROM alpine:3.13
 
-FROM debian:stretch
-WORKDIR /opt/meteor
-COPY --from=builder /build/meteor /opt/meteor/meteor
-RUN apt-get update
-RUN apt-get install -y --no-install-recommends ca-certificates
+COPY meteor /usr/bin/meteor
 
 CMD ["./meteor"]
```
Comment (StewartJingga) [inline Dockerfile:5]: use ["meteor"] instead

Comment (StewartJingga) [reply Dockerfile:5]: also dont forget to install ca-certificates manually on alpine

**Review by StewartJingga — APPROVED** [review]

**Comment by ravisuhag** [comment]
@StewartJingga @GrayFlash Build is failing on main for this. Dockerfile issue.

**Comment by StewartJingga** [comment]
Oh sorry I missed it during review.

@GrayFlash we need the goreleaser action with --snapshot flag to build the binary on `build` workflow.

**Comment by GrayFlash** [comment]
@StewartJingga @ravisuhag  missed that part, working on `goreleaser-config` branch temporarily, will create PR once it successfully gets triggered there.

**Pull request merged**